spec/models/converters/school_rating_converter_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SchoolRatingConverter do
+  subject { described_class }
```

Comment:
also not using `subject` in this test

Reply:
I can remove line 6 and just use described_class, but in order to use subject explicitly we'd have to disable the rubocop RSpec/NamedSubject rule or it'd throw a linter error. If you think that's a better solution I can make a separate PR to update .rubocop.yml, but I'd rather not have this wait on that if possible.